assets/dash.css

```diff
@@ -681,8 +681,8 @@ header .lang .dropdown .wpml-ls a {
   .halfbanner .carousel .image {
     background-size: cover;
     background-position: center;
-    height: 285px;
-    width: 100%;
+    height: 340px;
+    width: 95%;
```

Comment:
What does this change achieve? I think it is related to the news/blog page, right? In that case it should be in a separate PR.

Reply:
I left a comment with the change but I don't know where it went. I tried to make it separate but it auto merged it with the other open pull request. Haha! This changes the proportions of the featured blog post image so that it is closer to the dimensions of the featured post images so it doesn't crop them awkwardly.

Reply:
Can you revert this change and put it in a separate PR with an appropriate commit message please :+1: